README.md

```diff
@@ -1,78 +1,203 @@
-```  
-   git clone https://github.com/Javivy/react-capstone.git
-```   
-4. Open the project with your IDE. I use [Visual Studio Code](https://code.visualstudio.com/download)
-5. Run `npm install` to get the dependencies
-
-   not sure how to use npm? Read [this](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm).
-6. Run webpack:
-- Production:
-   ``` 
-      npm run build
-   ```  
-- Development:
-  ```
-   npm run start
-  ```
-  not sure how to use npm? Read [this](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm).
+4. npm install to install the needed packages.
+5. npm run start to see the website.
 
 ### Prerequisites
+
+In order to run this project you need:
 - Having an IDE .
 - Having Git installed and configured.
 
 ### Setup
-- Clone the repository
-- Open the folder with your IDE
-- Get the dependencies with `npm install` 
-  - not sure how to use npm? Read [this](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm).
 
-## Author
+Clone this repository to your desired folder:
+
+1. open terminal.
+2. cd <<Desired Folder>>
+- git clone https://github.com/Mhamad-Raad/Capstone-React.git
+
+
+### Install
+
+Install this project with:
+1. cd Capstone-React
+2. npm install
+
+### Usage
+
+To run the project, execute the following command:
+
+- npm run start.
+
+### Run tests
+
+To run tests, run the following command:
+
+- npm run test
+
+### Deployment
+
+You can deploy this project using:
+
+- Go into the root directory of this folder.
+- run npm run build.
+- manually deploy on netlifly, [tutorial](https://www.youtube.com/watch?v=tVzpC5_AC8M).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
 
 👤 **Javivy**
 
 - GitHub: [@Javivy](https://github.com/Javivy)
 - LinkedIn: [Javier Vargas](https://www.linkedin.com/in/javier-alejandro-vargas-ortega)
 - Twitter [@Javivy_2803](https://twitter.com/Javivy_2803)
 
-## 🤝 Contributing
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **[Add_Responsiveness]**
+- [ ] **[Add_Search_Functionality]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
 
 Contributions, issues, and feature requests are welcome!
 
 Feel free to check the [issues page](https://github.com/Javivy/react-capstone/issues).
 
-## Show your support
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+feel free to give it a star if you like it ⭐️.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Give credit to everyone who inspired your codebase.
```

Comment:
- Hey, please let's give appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/) to make our readme more professional